pkg/kubelet/dockershim/docker_container.go

```diff
@@ -243,10 +239,12 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 			glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
 		}
 
-		if dockerLoggingDriver == dockerDefaultLoggingDriver {
-			glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
-		} else {
-			glog.V(5).Infof("Unsupported logging driver: %s", dockerLoggingDriver)
+		for _, driver := range criSupportedLogDrivers {
+			if dockerLoggingDriver == driver {
+				glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
```

Comment:
@timstclair Thanks for your reviewing and comments. IMO, when we find realPath empty, we just need to give users some logs for debugging. We don't want to return error for any conditions when realPath is empty. Former discussion in #38691 with @Random-Liu and other folks.